app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,38 @@
+.app-c-email-signup {
+  @include govuk-responsive-margin(8, "top");
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-email-signup--full-width {
+  margin-top: 0;
+  border-left: 0;
+  border-right: 0;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.app-c-email-signup--transparent_background {
+  background: none;
+  border: none;
+}
+
+.app-c-email-signup--no-intro .govuk-link {
+  @include govuk-font(19, $weight: bold);
+  display: block;
+  margin-left: govuk-spacing(5);
+  margin-bottom: 0;
+}
+
+.app-c-email-signup__icon {
+  position: relative;
+  float: left;
+  margin-right: .5em;
```

Comment:
I'm not sure this is needed? Seems to add only a tiny amount of spacing, and the link to the right of it already has left margin, so if we do need that extra bit of space I think it would be better applied in a single place than multiple.